torch/ao/quantization/quantizer/xnnpack_quantizer.py

```diff
@@ -114,7 +115,7 @@ def get_symmetric_quantization_config(
         if is_dynamic:
             act_observer_or_fake_quant_ctr = PlaceholderObserver  # type: ignore[assignment]
         else:
-            act_observer_or_fake_quant_ctr = HistogramObserver  # type: ignore[assignment]
+            act_observer_or_fake_quant_ctr = MinMaxObserver if ptq_act_use_minmax_observer else HistogramObserver  # type: ignore[assignment]
```

Comment:
- this doenst seem like the right way to fix this. Also can you explain the underlying issue? If the scalar tensor didnt have float max, but some other value, would that work with histogram observer?

- Orthogonally, why does histogram observer not work with single value or float max?

- For fixing this, I would suggest that we specifically change observer only for nodes/edge, that you are trying to annotate, that has a scalar tensor. And not do that in a blanket manner.

Reply:
* histogram observer is using torch.histc, for scalar tensor, it only works up to some values, not to float_max, the largest that it can work is around float_max/10e15 or something like that
* probably related to histc implementation, I didn't look closely on this
* sure, I think that would require in the user side to just walk through the graph and annotate the nodes before applying global xnnpack quantizer, but this configuration is still needed I think?

Reply:
> * histogram observer is using torch.histc, for scalar tensor, it only works up to some values, not to float_max, the largest that it can work is around float_max/10e15 or something like that

Jerry, which op's quantization, via scalar to tensor pass, is triggering this?

Reply:
this is mul/add, both triggers this I think

Reply:
> sure, I think that would require in the user side to just walk through the graph and annotate the nodes before applying global xnnpack quantizer, but this configuration is still needed I think?

It shouldnt be done by user. xnnpack quantizer should be able to do this in its annotation logic